code/ui/lisibilite_ui/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for lisibilite_ui project.
+
+Generated by 'django-admin startproject' using Django 3.0.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.0/ref/settings/
+"""
+
+import os
+from django.conf import settings
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'hc)l7zu(fg*o_arf2(&&wj-)mkyr_j65+^#r5v*&b%6k*or0=1'
```

Comment:
in future we have to move this to a env.py sort of file

Reply:
Agreed. Need to check how many such items need to be moved.